todo_app/app.py

```diff
@@ -1,11 +1,50 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect
 
+from todo_app.data.session_items import (
+    get_items,
+    add_item,
+    remove_item,
+    get_item,
+    save_item,
+)
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
-@app.route('/')
+@app.route("/")
 def index():
-    return 'Hello World!'
+    completed_items = []
+    incomplete_items = []
+    for item in sorted(get_items(), key=lambda item: item["id"]):
+        if item["status"] == "Completed":
+            completed_items.append(item)
+        else:
+            incomplete_items.append(item)
```

Comment:
Minor, I think a list comprehension looks a bit nicer here

```suggestion
    items = get_items()
    completed_items = [item for item in items if item["status"] == "Completed"]
    incomplete_items = [item for item in items if item["status"] != "Completed"]
```